packages/react/package.json

```diff
@@ -302,7 +302,7 @@
     "eslint-plugin-prettier": "3.4.1",
     "eslint-plugin-react": "7.26.1",
     "eslint-plugin-react-hooks": "1.7.0",
-    "eslint-plugin-testing-library": "3.10.2",
+    "eslint-plugin-testing-library": "3.9.2",
```

Comment:
3.9.2 instead of 3.10.2, because 3.10.2 has a bug that pops up in vscode all the time related to a new rule they added. A bump to v4 where it's fixed requires upgrade eslint and opens a whole other can of worms.